### PR TITLE
fix(agile best practices): updated user stories

### PR DESCRIPTION
**Pull request by homelessbirds**
Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

File: src/content/docs/agile-handbook/appendices/ticket-best-practices.mdx
Comment (austin-schaefer) [inline src/content/docs/agile-handbook/appendices/ticket-best-practices.mdx:99]: Is the `*` here meant for a bullet point? If so, I think you'll need a space between the `*` and `According`.

Comment (barbnewrelic) [inline src/content/docs/agile-handbook/appendices/ticket-best-practices.mdx:99]: If you look at your Gatsby preview https://docswebsitedevelop-sdagileuserstories.gtsb.io/docs/agile-handbook/appendices/ticket-best-practices/#jira_checklist , you will see that this is not rendering quite right.

Comment (austin-schaefer) [inline src/content/docs/agile-handbook/appendices/ticket-best-practices.mdx:99]: * I think there should be a comma after `According to Atlassian` before beginning the quote
* Looks like closing quote mark is missing

File: src/content/docs/agile-handbook/appendices/ticket-best-practices.mdx
Comment (austin-schaefer) [inline src/content/docs/agile-handbook/appendices/ticket-best-practices.mdx:100]: * Is this continuing the quote from before? If so, I think you probably want a separate set of quotes. 
* This closes with `'"'` I think at least one of those quoets might be extraneous?

Comment (barbnewrelic) [inline src/content/docs/agile-handbook/appendices/ticket-best-practices.mdx:100]: I don't understand why this line ends with '"' Is that intentional?
Also, if you want lines 99 and 100 to be bullets, there should be a space after the * beginning those lines. Hope this helps!

File: src/content/docs/agile-handbook/appendices/ticket-best-practices.mdx
Comment (austin-schaefer) [inline src/content/docs/agile-handbook/appendices/ticket-best-practices.mdx:175]: Same comment as above re:space between asterisk and start of bullet copy

File: src/content/docs/agile-handbook/appendices/ticket-best-practices.mdx
Comment (austin-schaefer) [inline src/content/docs/agile-handbook/appendices/ticket-best-practices.mdx:176]: Missing quote mark between `would be:` and `A complete`?

**Review by austin-schaefer — DISMISSED** [review]

File: src/content/docs/agile-handbook/appendices/ticket-best-practices.mdx
Comment (barbnewrelic) [inline src/content/docs/agile-handbook/appendices/ticket-best-practices.mdx:175]: Same comment here re: * at beginning of line.

File: src/content/docs/agile-handbook/appendices/ticket-best-practices.mdx
Comment (barbnewrelic) [inline src/content/docs/agile-handbook/appendices/ticket-best-practices.mdx:176]: If I am reading this correctly, I think you need a " before A complete

**Review by barbnewrelic — DISMISSED** [review]
Free to merge this PR after you decide what to do with my questions and comments, take 'em or leave 'em! I don't need to do a second review. Thanks for updating the agile handbook!

File: src/content/docs/agile-handbook/appendices/ticket-best-practices.mdx
Comment (homelessbirds) [reply src/content/docs/agile-handbook/appendices/ticket-best-practices.mdx:100]: nope! not intentional! thank you!

**Review by zuluecho9 — APPROVED** [review]

**Pull request merged**
